src/braket/experimental/autoqasm/operators/exceptions.py

```diff
@@ -0,0 +1,28 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+"""Operators for exception handling."""
+
+from typing import Callable
+
+
+def assert_stmt(test: bool, message: Callable) -> None:
```

Comment:
So this passes through Python asserts in our programs? What happens if I try `assert measure(0);`?

Reply:
Yes, this is only for Python asserts (which are evaluated at trace time). I'll add an explicit check/exception here if a user tries to assert on a measurement or AutoQASM variable.